bin/kafka-producer-perf-test.sh

```diff
@@ -17,4 +17,4 @@
 if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
-exec $(dirname $0)/kafka-run-class.sh kafka.tools.ProducerPerformance $@
+exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.clients.tools.ProducerPerformance $@
```

Comment:
Currently core project uses jopt-simple jar and tools project uses argparse4j jar.  argparse4j is used in VerifiableProducer.  I think we should remove one of the dependency. Can I remove argparse4 dependency and use jopt-simple in tools project?  cc @@junrao  @granders